src/Clan.ts

```diff
@@ -245,4 +298,73 @@ export class Clan {
     );
     return result.includes("You contributed");
   }
+
+  /**
+   * Take items from the stash
+   * 
+   * This function will also take equivalent foldables if the original item cannot be found
+   * 
+   * @param items Items to take
+   * @returns Items successfully taken
+   */
+  @validate
+  take(items: Item[]): Item[] {
+    return items.filter((item) => {
+      if (have(item)) return false;
+
+      const foldGroup = getFoldGroup(item);
+
+      if (foldGroup.some(fold => have(fold))) {
+        cliExecute(`fold ${item.name}`);
```

Comment:
I know this part comes straight from me, but is there a nicer way to achieve this than cliExecute?

Reply:
Nope!